test-locally.sh

```
@@ -6,12 +6,24 @@ set +m
+SOLANA_STARTED_BY_US=false
+EPHEMERAL_STARTED_BY_US=false
 PASSED_TESTS=()
 FAILED_TESTS=()
 FAILED_TESTS_NAMES=()
 FAILED_TESTS_ERRORS=()
 TEST_COUNT=0
 
+# Check if a port is in use
+check_port() {
+  local port=$1
+  if lsof -Pi :$port -sTCP:LISTEN -t >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)